client/components/mma/Page.tsx

```diff
@@ -18,20 +24,33 @@ export interface PageContainerProps {
 	selectedNavItem: NavItem;
 	pageTitle: string | ReactElement;
 	compactTitle?: boolean;
+	minimalFooter?: boolean;
 }
 
-export const PageContainer = (props: PageContainerProps) => (
-	<>
-		<PageHeaderContainer
-			selectedNavItem={props.selectedNavItem}
-			title={props.pageTitle}
-			compactTitle={props.compactTitle}
-		/>
-		<PageNavAndContentContainer selectedNavItem={props.selectedNavItem}>
-			{props.children}
-		</PageNavAndContentContainer>
-	</>
-);
+export const PageContainer = (props: PageContainerProps) => {
+	const hasMinimalFooterContext = useContext(
+		HasMinimalFooterContext,
+	) as HasMinimalFooterInterface;
+
+	useEffect(() => {
+		hasMinimalFooterContext.setHasMinimalFooter(
```

Comment:
No change required but this feels like another point to needing better state management... 🐘